1-js/05-data-types/02-number/article.md

```diff
@@ -320,7 +320,7 @@ let num = +prompt("Enter a number", '');
 alert( isFinite(num) );
 ```
 
-Please note that an empty or a space-only string is treated as `0` in all numeric functions including `isFinite`. 
+Please note that spaces in a string are treated as a `0` in all numeric functions, including `isFinite`. 
```

Comment:
That's wrong: the spaces are not treated as `0`.

An empty string is `0`.
A space-only string is `0`.

But spaces are generally not `0`, like here: "1 2" -- invalid number (not 102).

Reply:
Agree with @iliakan . Original content is more accurate.